locales/ar/learn.json

```diff
@@ -12,7 +12,7 @@
   "contributors": "المساهمون",
   "day": "اليوم",
   "description": "الوصف",
-  "duration-daily": "يستغرق كل درس قصير أقل من {{dailyMins}} دقيقة لإكمال {{days}} دروس إجمالية",
+  "duration-daily": "يستغرق كل درس قصير أقل من {{dailyMins}} دقيقة لإكمال {{days}} دروس إجمالية.",
```

Comment:
Why don't we just add `.` after we use the translation value `duration-daily` itself like this? 

```
description={`${t('duration-daily', {
          dailyMins: dailyMinutes,
          days: lessons.length,
        })}.`}
```

Also if we are adding/update translations, we should only update `en` file.